doc/nrf/releases/release-notes-2.4.1.rst

```diff
@@ -0,0 +1,106 @@
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This minor release adds changes on top of the :ref:`nRF Connect SDK v2.4.0 <ncs_release_notes_240>`.
+The changes affect <TBD>
+
+See :ref:`ncs_release_notes_241_changelog` for the complete list of changes.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.4.1**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.4.1`_.
+
+IDE and tool support
+********************
+
+`nRF Connect for Visual Studio Code extension <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.4.1.
+SEGGER Embedded Studio Nordic Edition is no longer tested or recommended for new projects.
+
+:ref:`gs_app_tcm`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.4.1_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.4.1_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.4.1`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_241_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+nRF IEEE 802.15.4 radio driver
+------------------------------
+
+* Added a workaround for known issue KRKNWK-16976 allowing to detect crashes of the nRF5340 network core running the 802.15.4 stack.
+
+Apple Find My
+-------------
+
+* Fixed two issues in the Apple Find My add-on for the |NCS|.
+  See the detailed description in the Find My release notes.
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``b57a7488aa3a49fe71b9a042a3d54f2d463b352b``.
```

Comment:
This SHA should be the same as for v2.4.0 - we did not upmerge Zephyr on the release branch (so `4bbd91a9083a588002d4397577863e0c54ba7038`).